L1Trigger/TrackFindingTracklet/python/Producer_cff.py

```diff
@@ -0,0 +1,14 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
I suggest renaming this ProducerKF_cff.py (and similarly for _cfi.py), since otherwise its name gives no clue which step of Hybrid algo it is.

Reply:
done